core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -366,8 +363,8 @@ contract Liquidatable is PricelessPositionManager {
 
         // There are three main outcome states: either the dispute succeeded, failed or was not updated.
         // Based on the state, different parties of a liquidation can withdraw different amounts.
-        // Once a caller has been paid their address deleted from the struct.
-        // This prevents them from being paid multiple from times the same liquidation.
+        // Once a caller has been paid their address is deleted from the struct.
+        // This prevents them from being paid multiple times from the same liquidation.
```

Comment:
Change to more clear: "This prevents paying the caller multiple times from the same liquidation"